app/controllers/api/v2/compute_resources_controller.rb

```diff
@@ -90,27 +90,31 @@ def available_images
       param :id, :identifier, :required => true
       def available_clusters
         @available_clusters = @compute_resource.available_clusters
+        @total = @available_clusters.try(:size).to_i
```

Comment:
Nice, but: `nil.to_i` will return zero. Can we change this to turn nil if size is not available?

Reply:
I was thinking of something like this:

```suggestion
        @total = @available_clusters.respond_to?(:size) ? @available_clusters.size.to_i : nil
```

Reply:
I thought it is a desired output :) But nil is probably better.
Wouldn't be enought just remove to_i ?

Reply:
I'd be ok with that as well, but I don't like `.try`. It swallows all errors and makes it hard to debug.

Reply:
Yeah, understood. How about &. operator? That's what we actually wants, right?

Reply:
I believe it has the same issue as `.try`, but I it's fine this way. Thanks again!